client/components/OrganisationCard.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
     organisation: Organisation
 }
 
-const OrganisationCard = ( { organisation }: Props ) => {
+function OrganisationCard( { organisation }: Props ) {
```

Comment:
How come you changed it to function?

Reply:
I just wanted to check whether it would work the same way in React

Reply:
Fair, it essentially does, but arrow functions are preferred usually unless you have to.

https://www.geeksforgeeks.org/arrow-functions-in-javascript/